src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -105,6 +114,11 @@
       <td>
         <mc-time-ago [value]="node.build.dateProduced"></mc-time-ago>
       </td>
+      <td>
+        <fa-icon [icon]="hasCycle(node) ? 'redo' : 'exclamation-triangle'" 
```

Comment:
You don't need the `hasCycle` and `cyclePath` functions. You can just put `node.hasCycles` and `node.cyclePath` in the html.